.gitignore

```diff
@@ -18,3 +18,6 @@ grounded-*
 # macOS
 .DS_Store
 yarn.lock
+
+# Ignore unused code file
+UnUsedCode.tsx
```

Comment:
You shouldn't keep UnUsedCode, if it's unused, just remove it (from your clone of the repo and from .gitignore). 
I like the idea of cleaning up the unused code by deleting it. Thanks for keeping things clean.

Reply:
sure, Dr. Holdener, I will remove it.